src/vendor/wordpress/styles/_jetpack-blocks.scss

```diff
@@ -1,5 +1,139 @@
+@use '../../../mixins/border-radius';
+@use '../../../mixins/headings';
+@use '../../../mixins/input';
 @use '../../../mixins/spacing';
+@use '../../../mixins/theme';
+@use '../../../mixins/toggle-input';
+@use '../../../compiled/tokens/scss/color';
+@use '../../../compiled/tokens/scss/size';
+@use '../../../compiled/tokens/scss/font-weight';
 
+/// Apply consistent vertical rhythm to Jetpack blocks.
+.wp-block-jetpack-contact-form,
 .wp-block-jetpack-markdown {
   @include spacing.vertical-rhythm;
 }
+
+/// For some reason the Jetpack contact form adds `16px` of padding as an inline
+/// style with no mechanism for customization. This is a hack to remove it.
+.wp-block-jetpack-contact-form {
+  padding: 0 !important;
+}
+
+/// Apply our existing form component styles to Jetpack form fields.
+.grunion-field {
+  /// Test-based inputs
+  &:not([type='checkbox']):not([type='radio']) {
+    @include input.default;
+  }
+
+  /// Checkboxes
+  &[type='checkbox'] {
+    @include toggle-input.checkbox;
+  }
+}
+
+/// Apply the same spacing to fields and labels as in our Form Group object.
+.grunion-field-wrap,
+.grunion-checkbox-multiple-options,
+.grunion-radio-options {
+  display: grid;
+  gap: #{size.$spacing-gap-form-group-default};
+}
+
+/// Style the "(required)" text that follows required field labels.
+.grunion-field-label > span {
+  color: var(--theme-color-text-muted);
+  margin-inline-start: 1ch;
+}
+
+/// Arrange the label and input for checkboxes and radio buttons side-by-side.
+.grunion-field-label.checkbox,
+.grunion-checkbox-multiple-label,
+.grunion-radio-label {
+  display: grid;
+  gap: 1ch;
+  grid-template-columns: auto 1fr;
+}
+
+/// Style consent messaging fields in a visually distinct way, similar to a card
+/// or contained WordPress block with a background.
+.grunion-field-consent-wrap {
+  @include border-radius.conditional;
+  @include spacing.fluid-margin-inline-negative;
+  @include spacing.fluid-padding-inline;
+  background-color: var(--theme-color-background-secondary);
+  display: block;
+  padding-block: size.$rhythm-default;
+}
+
+/// The submission message markup seems _ancient_. It apppears to be using
+/// empty paragraphs to create spacing between elements. Luckily we're able to
+/// style things adequately in spite of this.
```

Comment:
I wonder if we should be adding `p:empty { display: none }` to yoink those empty paragraphs out of the a11y tree?

Reply:
Good idea!

Reply:
@Paul-Hebert I've made this change (see e733007), but it required refactoring the way the layout of this CSS works (since the hidden paragraphs no longer allowed the `inline-block` elements to break). I think this is better from the end user's perspective, just FYI when it comes time to re-review!